packages/core/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@manypkg/core",
```

Comment:
I don't exactly like the `/core` in this name - I don't necessarily have a better name for this just yet though. The boundary line between core/get-packages/find-root is quite blurry to me at the moment. 

Maybe something like `@manypkg/tools` (`@manypkg/project-tools`) would work for this? To pull off smth like this, we'd probably have to remove `expandPackageGlobs` from the public exports of this package though (which doesn't sound to be bad, the `getPackages` methods already call this so it's not necessary to export this for public consumption)

Reply:
:+1: Also somewhat agnostic on the name, and totally fine if expandPackageGlobs is just some helper method for tool authors and not a public export.

When thinking about the name, though, consider this line of reasoning (which is still only half-baked at the moment)...

 - I'd eventually like `changesets` to be usable even in a monorepo of a different language, like Swift or Kotlin.
 - In these worlds, the file to "read" and "write" might be a totally different format, like a `project.yaml` file or a `gradle.properties` file.
 - In fact the act of "writing" a new version might be totally unrecognizable, perhaps it involves reading or writing several different files for one project.
 - To allow a tool author to contribute such a thing, the surface area of Tool needs to eventually grow to encompass more of what changesets does... the `PackageJSON` object perhaps morphs into a more agnostically named `Project` object, but still requires a name, version, dependency list, etc.; the `Tool` interface grows to include `updateProject(version: SemVer)` or something similar.
 - Some systems allow major, minor, patch, some also semver concepts like -alpha, -prerelease, -snapshot, etc., so a Tool might need to specify what's allowed. In fact, both Swift and Kotlin have complicated relationships with version, because those projects both have "libraries" (released as CocoaPods or Swift Packages, or to Maven) that support full semver spec, but their "applications" (released to App Store or Google Play) are much more explicit and restricted; so there's even a "type" of project we might eventually want to model.

If more and more "functionality" ends up inside Tools, the name `tools` still makes sense so we're probably OK. This also isn't necessarily the right direction (if we decide writing a version and calculating new versions are too similar, then perhaps you add new Tool to manypkg but Changesets still has to add explicit support for that tool in its own list of tool implementations).

Reply:
With all that in mind - `/tools` still makes sense and I would probably go with that. I'm also not that afraid of revisiting the name in the future. It's just that `/core` is such generic that it's hard to tell what it even does.